presto-main/src/test/java/io/prestosql/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -482,6 +483,17 @@ public CorrelatedJoinNode correlatedJoin(List<Symbol> correlation, PlanNode inpu
         return new CorrelatedJoinNode(idAllocator.getNextId(), input, subquery, correlation, type, filter, originSubquery);
     }
 
+    public TableScanNode tableScan(List<Symbol> symbols, boolean forDelete)
```

Comment:
nit: you could probably extend some exiting method and add an override, but it's also fine

Reply:
I decided to add a separate method for now. I think we could reorganize the `tableScan()` methods later when `forDelete` property is more used and tested.